ParquetClassLibrary/Crafting/StrikePanel.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using ParquetClassLibrary.Utilities;
+
+namespace ParquetClassLibrary.Crafting
+{
+    /// <summary>
+    /// Models the panels that the player must strike during item crafting.
+    /// </summary>
+    public struct StrikePanel
+    {
+        /// <summary><c>true</c> if this panel is not used in the current crafting recipe; otherwise, <c>false</c>.</summary>
+        public bool IsVoid { get; set; }
+
+        /// <summary>
+        /// The range of values this panel can take on while being worked.  <see cref="Range{T}.Minimum"/> is normally 0.
+        /// </summary>
+        public Range<int> WorkingRange { get; set; }
```

Comment:
nit: What if I made a StrikePanel with a valid WorkingRange and IdealRange, but then changed the WorkingRange? Maybe these values could be readonly?

Reply:
Excellent point.